docs/docs/create-source-plugin.md

```diff
@@ -89,10 +89,7 @@ But at a high-level, these are the jobs of a source plugin:
 - "Link" nodes types you create as appropriate (see
   [_Node Link_](/docs/api-specification/) in the API specification concepts
   section.
-- Return either a promise or use the callback (3rd parameter) to report back to
-  Gatsby when you're done sourcing nodes. Otherwise either Gatsby will continue
-  on before you're done sourcing or hang while waiting for you to indicate
-  you're finished.
+- Return either a promise or use the callback (3rd parameter) to report back to Gatsby when `sourceNodes` is fully executed. If a promise or callback isn't returned, Gatsby will continue on in the build process, before nodes are finished being created. Your nodes might not end up in the generated schema at compilation, or the process will hang while waiting for an indication that it's finished.
```

Comment:
Was I right to format `sourceNodes` as code? We format all words that could be used as code.

Also, now I realize the last sentence in this paragraph doesn't make much sense. Any ideas on how to edit that?

Reply:
Also hello! I remember we got to meet last week!

Reply:
👋 @shannonbux yes it was great meeting you! Nice change on `sourceNodes`. Would it be best to drop the last sentence?

Reply:
I'll leave it for now and merge this change! Thanks!

Reply:
Excellent, thank you!